src/test/java/com/serenitydojo/WhenCreatingObjects.java

```diff
@@ -4,4 +4,21 @@
 import org.junit.Test;
 
 public class WhenCreatingObjects {
+
+    @Test
+    public void creating_a_dog() {
```

Comment:
could you add a `createAnotherDog` test with 

```
    Dog fido = new Dog("Fido","Bone", 5,"Woof", false);
    Assert.assertEquals(fido.getName(), "Fido");
    Assert.assertEquals(fido.getFavoriteToy(), "Bone");
    Assert.assertEquals(fido.getAge(), 5);
    Assert.assertEquals(fido.makeNoise(), "Woof");
    Assert.assertEquals(fido.feed(), false);
```

and find out why it fails? :)

Reply:
also 👍 for using assert :)

Reply:
I have added a new test  **createAnotherDog** 
Thanks